packages/blade/src/components/Input/BaseInput/StyledBaseInput.native.tsx

```diff
@@ -181,6 +187,10 @@ export const StyledBaseInput = React.forwardRef<
           setCurrentInteraction('active');
         }}
         onBlur={(): void => {
```

Comment:
can we move the common props out in an object? its very difficult when you add/remove things from here as someone need to be very cautious to add/remove  from styled button as well as styled input

Reply:
I did that for web file. In native, there are just 2 props which are same. Rest need a bit of separate handling because one is input where we read `nativeEvent.text` for value and one is button where we read the inner text content for the value.

Reply:
moved these 2 props that are same